src/api/Gemfile.lock

```diff
@@ -1,3 +1,14 @@
+GIT
```

Comment:
There is already a new release, but the travis support is not working properly. Travis behave different in a PR than in a branch. In the branch works, so I need the PR to find out was going on.